docs/upgrade.html

```diff
@@ -43,6 +43,10 @@ <h5><a id="upgrade_360_notable" href="#upgrade_360_notable">Notable changes in 3
             See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-925%3A+Rack+aware+task+assignment+in+Kafka+Streams">KIP-925</a>
             and <a href="/{{version}}/documentation/streams/developer-guide/config-streams.html#rack-aware-assignment-strategy"><b>Kafka Streams Developer Guide</b></a> for more details.
         </li>
+        <li>To account for a break in compatibility introduced in version 3.1.0, MirrorMaker 2 has added a new
+            <a href="#mirror_connector_replication.policy.internal.topic.separator.enabled">replication.policy.internal.topic.separator.enabled</a>
+            property. If upgrading from 3.0.x or earlier, it may be necessary to set this property to <code>false</code>; see the property's
+            <a href="#mirror_connector_replication.policy.internal.topic.separator.enabled">documentation</a> for more details.</li>
```

Comment:
This second link feels redundant.

Reply:
Is there a disadvantage to it?

Reply:
It may just be my personal preference then. I'm used to the "wikipedia style" (i don't know what else to call it) where a link is provided the first time something is mentioned and then not afterwards. When I see two links adjacent to one another, I expect that they will go somewhere different.

This is not a blocking issue.

Reply:
> When I see two links adjacent to one another, I expect that they will go somewhere different.

That's a fair point. I'm a little hesitant to only include a link in the first mention of the property, since people may still not think to click the property to see docs for it. I've modified the first mention of it to use a `<code>` block instead, and kept the second link where it was. How does that look to you?